charts/refractr/templates/issuer.yaml

```diff
@@ -18,4 +18,20 @@ spec:
       - http01:
           ingress:
             class: nginx
+      - dns01:
+          # This is so we can instruct LE to follow a CNAME on the target domain to
+          # a domain we control (refractr.mozit.cloud) in route53
+          cnameStrategy: Follow
+          route53:
+            region: us-west-2
+            hostedZoneID: Z06469063HC60S7SHF706
+            role: "arn:aws:iam:::role/itse-apps-stage-1-cert-manager-role"
```

Comment:
Does the role need to live in the manifest? Shouldn't cert-manager itself just have that permission? Wouldn't annotating the service account itself with the role be more than enough.

See https://aws.amazon.com/blogs/opensource/introducing-fine-grained-iam-roles-service-accounts/ 

If annotating the service account doesn't work you should templatize it